src/vcpkg/build.cpp

```diff
@@ -309,7 +309,26 @@ namespace vcpkg::Build
         "",
         "Error: in triplet {triplet}: Unable to find a valid toolchain combination.\n    The requested target "
         "architecture was {value}\n    "
-        "The selected Visual Studio instance is at {path}\n    The available toolchain combinations are {list}\n");
+        "The selected Visual Studio instance is at {path}\n    The available toolchain combinations are {list}\n"
+        "Please check the required Visual Studio Components:\n"
+        "C++:\n"
+        "   Windows Universal C Runtime\n"
+        "   C++ core desktop features\n"
+        "   Visual Studio Build tools:\n"
+        "       * Microsoft.VisualStudio.Component.VC.140 (for Visual Studio 2015)\n"
+        "       * MSVC v141/v142/v143 - VS 2017/2019/2022 C++ x64/x86 build tools (for Visual Studio 2017 or later)\n"
+        "   MSBuild\n"
+        "Windows SDK:\n"
+        "   * 8.1\n"
+        "   * 10.0.18362\n"
```

Comment:
Wait, do we actually need all these independent copies of the Windows SDK? That seems wrong.

Reply:
Maybe I should write an explanation here, everything starting with an asterisk is optional.

Reply:
Or write `One of the following Windows SDKs:`